xformers/components/attention/csrc/cuda/attention.cu

```diff
@@ -473,56 +497,30 @@ __global__ void attention_kernel(
       output_block[q_item_idx][k] = tmp;
     }
   }
+
+  if (compute_logsumexp) {
+#pragma unroll
```

Comment:
guess is that the pointer array is because of this ?

Reply:
I think it might also help as well, yes, although it was not directly the main reason why I did it